Foundation/NSPathUtilities.swift

```diff
@@ -446,57 +446,108 @@ public extension NSString {
     /// - Experiment: This is a draft API currently under consideration for official import into Foundation
     /// - Note: Since this API is under consideration it may be either removed or revised in the near future
     public func completePathIntoString(inout outputName: NSString?, caseSensitive flag: Bool, inout matchesIntoArray outputArray: [NSString], filterTypes: [String]?) -> Int {
-        // FIXME: I guess, it should be NSURL(fileURLWithPath: _storage), but it is not implemented yet.
         guard !_storage.isEmpty else {
             return 0
         }
         
-        guard let url = NSURL(string: _storage) else {
-            return 0
-        }
-        
-        let normalizedTypes = flag ? filterTypes : filterTypes?.map { $0.lowercaseString }
-        let types = Set<String>(normalizedTypes ?? [])
-        let compareOptions = flag ? [] : NSStringCompareOptions.CaseInsensitiveSearch
+        let url = NSURL(fileURLWithPath: _storage)
```

Comment:
this probably shouldn't directly access _storage since it may be a NSCFString or a subclass, so _swiftObject is the better accessor here